mmedit/datasets/pipelines/crop.py

```diff
@@ -532,3 +533,49 @@ def __call__(self, results):
             raise ValueError(f'Wrong img ndim: {img.ndim}.')
         results['gt'] = img
         return results
+
+
+@PIPELINES.register_module()
+class CropLike:
+    """Crop/pad a key according to the size of another key.
```

Comment:
Crop/pad the image in the target_key according to the size of image in the reference_key .